README.md

```diff
@@ -145,6 +145,8 @@ const options = {
     // You can also set "latest" to use the most recently supported version.
     ecmaVersion: 5,
 
+    allowReserved: true, // only allowed when ecmaVersion is 3
+
```

Comment:
After adding the line, the example won't work.  Should the above ecmaVersion be changed to 3?

Reply:
sure, happy to do that

Reply:
hm, below, there's a comment `enable implied strict mode (if ecmaVersion >= 5)` - will that throw with ecmaVersion of 3?

Reply:
no, it won't throw:
![image](https://user-images.githubusercontent.com/13050025/142968709-7a443047-5d57-435c-b37c-b32df492fb32.png)

Reply:
hmm - given that precedent, should `allowReserved` work on every ecmaVersion, not just 3?

Reply:
In my understanding, it is only allowed to be `false` when ecmaVersion > 3.

Reply:
hm, the code in acorn seems to allow it for every ecmaVersion - this is the code in Parser:
```js
if (options.allowReserved !== true) {
  reserved = reservedWords[options.ecmaVersion >= 6 ? 6 : options.ecmaVersion === 5 ? 5 : 3];
  if (options.sourceType === "module") { reserved += " await"; }
}
```
which implies that setting it to `true` can work for any ecmaVersion, as does `never`.

Reply:
yes, but we want to limit it in espree, as seen here: https://github.com/eslint/tsc-meetings/pull/301/files#diff-38e15b82b50329417e488192e541c32153feb4e89c73e191845eb9d988587779R29

Reply:
Alrighty, i'll update the readme.